applications/callflow/src/module/cf_ring_group.erl

```diff
@@ -24,14 +24,28 @@
 %%--------------------------------------------------------------------
 -spec handle(wh_json:object(), whapps_call:call()) -> 'ok'.
 handle(Data, Call) ->
-    case get_endpoints(Data, Call) of
+    Repeat = wh_json:get_value(<<"repeats">>, Data, 1),
```

Comment:
I'd use `wh_json:get_integer_value/3` to ensure Repeat is an integer (since `"repeats":"3"` could be stored).